Makefile

```diff
@@ -14,7 +14,7 @@ TESTS=	tests/lbs tests/kvlds tests/mux tests/s3 tests/kvlds-s3 \
 	perftests/s3 perftests/s3_put perftests/serverpool	\
 	perftests/dynamodb_sign perftests/dynamodb_request	\
 	perftests/dynamodb_queue perftests/dynamodb_kv		\
-	perftests/kvldsclean-ddbkv				\
+	perftests/kvldsclean-ddbkv perftests/https		\
```

Comment:
Am I correct in thinking that this is in `perftests/` instead of `tests/` because it should only be run manually?

Reply:
Yes.  Not that I have a problem with making automated requests to `https://www.google.com` but (like the `http` perftest) it's a "test runs and then a human looks at the output to make sure that it's correct" test.

But some of it could definitely move into `tests`, e.g. making sure that we can connect to good hosts and don't connect to *.badssl.com.  So it's just that the test *as it exists right now* needs human inspection of the output, not that all possible tests of this code would.

Reply:
I was more asking because of where I should put a `network_ssl/` test.  If we might be adding other APIs, I want a low-level test in place first.  `perftests/` seems fine.

Reply:
I think it would be good to have a test in `/tests/` which uses `network_ssl` code to open connections to port 443. send a single byte, then drop the connection and exit.  If the code works properly, the SSL handshake will occur (or fail) before `network_ssl_write`'s callback fires.